recipes/tobias/meta.yaml

```
@@ -1,54 +1,55 @@
-{% set name = "tobias" %}
-{% set version = "0.5.0" %}
-
-package:
-  name: {{ name|lower }}
-  version: {{ version }}
-
-source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: ae14018c6992c6e8f415e142ee8bbcb3b0932927203fe2061a6ac87169631fef
-
-build:
-  number: 0
-  skip: True # [py27 or osx]
-
-requirements:
-  host:
-    - {{ compiler('gcc') }}
-    - python
-    - setuptools
-    - cython
-    - numpy
-  build:
-    - numpy
-  run:
-    - python
-    - numpy
-    - scipy
-    - pysam
-    - matplotlib >=2
-    - scikit-learn
-    - pandas
-    - pypdf2
-    - xlsxwriter
-    - pyBigWig
-    - moods
-    - pybedtools
-    - adjusttext
-
-test:
-  commands:
-    - "TOBIAS --help"
-
-about:
-  home: https://github.molgen.mpg.de/loosolab/TOBIAS
-  doc_url: https://github.molgen.mpg.de/loosolab/TOBIAS/wiki
-  license: MIT
-  summary: |
-    TOBIAS (Transcription factor Occupancy prediction By Investigation of ATAC-seq Signal) is a collection of command-line 
-    bioinformatics tools for performing footprinting analysis on ATAC-seq data. 
-
-extra:
-  recipe-maintainers:
-    - msbentsen
+{% set name = "tobias" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2afda6d73bbfab2f1a77ec14a3780e54c5f2b7aa0cb1cb25194a3ece16ddfbf5
+
+build:
+  number: 0
+  skip: True # [py27 or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - setuptools
+    - cython
+    - numpy
+  run:
+    - python
+    - numpy
```

Looks like this links to `numpy`: https://github.molgen.mpg.de/loosolab/TOBIAS/blob/8cad568c9224c351bc02deb380a9d7d7958cd4ae/tobias/utils/ngs.pyx#L19
```suggestion
    - {{ pin_compatible('numpy') }}
```

---

Whoa, I didn't use that GitHub `suggestion` thing before!
Thanks for using that `Commit suggestion` button @jenzopr! I wouldn't have noticed that existed otherwise :D. Neat!